### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by jjqbxm** [anthonyweidai/SvANet#25]
Hi, using the publicly available dataset in data.md, manually dividing the dataset, and in doing the .sh file, doesn't it just run the results from the paper? 
Why is the result not correct, it just keeps showing overfitting

![Image](https://github.com/user-attachments/assets/aa676e0a-2396-4670-b2b1-c1128def34c7)

![Image](https://github.com/user-attachments/assets/bc545941-85fa-4dd5-9328-d304636ced49)

**Comment by anthonyweidai** [comment]
Someone mentioned the FIVES dataset. Actually, you should add split name before the mask and image since data from test and train sets may have the same name. Please find my answer in https://github.com/anthonyweidai/SvANet/issues/23#issuecomment-2870566663.

**Comment by jjqbxm** [comment]
![Image](https://github.com/user-attachments/assets/e92125c0-999d-471d-9c52-2a27a88c88a1)

Hi, as per your reply, I reconstructed my FIVES dataset naming, the result is still the same, and the training result is the same for a different dataset, besides, I didn't modify the program by myself, may I ask why is this?  
Thank you in advance for your reply!
